modules/lightning_telemetry/lightning_telemetry.module

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * Implements hook_cron().
+ */
+function lightning_telemetry_cron() {
+  /** @var \Drupal\lightning_telemetry\Telemetry $telemetry_service */
+  $telemetry_service = \Drupal::service('lightning_telemetry.telemetry');
+  $telemetry_service->sendTelemetry('Drupal cron ran');
+}
+
+/**
+ * Implements hook_modules_installed().
+ */
+function lightning_telemetry_modules_installed($modules) {
+  /** @var \Drupal\lightning_telemetry\Telemetry $telemetry_service */
+  $telemetry_service = \Drupal::service('lightning_telemetry.telemetry');
+  $installed_lightning_extensions = array_intersect($modules, $telemetry_service->getLightningExtensionNames());
```

Comment:
I think Lightning Core includes a class, called ComponentDiscovery, which is responsible for locating all Lightning components in a system. We should use that, rather than hard-coding a list into the telemetry service.